internal/models/v2models/environment.go

```diff
@@ -25,6 +26,7 @@ type Environment struct {
 	RequiresSuitability *bool
 	BaseDomain          *string
 	NamePrefixesDomain  *bool
+	HelmfileRef         *string `gorm:"not null; default:master"`
```

Comment:
Same comment as for cluster -- the default here is inconsistent with the controller and we should check that the field doesn't get set to empty